src/components/SummaryCard/SummaryCardBody/__tests__/SummaryCardBody.spec.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file Summary card body tests.
+ * @copyright IBM Security 2020
+ */
+
+import { render } from '@testing-library/react';
+import React from 'react';
+
+import { SummaryCardBody } from '../../../..';
+
+describe('SummaryCardBody', () => {
```

Comment:
Will we need a11y tests for these also?

Reply:
All tested together inside `SummaryCard`